### ISSUE TITLE
Homework 5 is ready for grading!

### ISSUE THREAD

**Issue by Maria815** [Maria815/STAT545-hw-Guo-Maria#4]
Thank you!

**Comment by rdwyer2** [comment]
## Factor Management
* You completed the necessary steps, but the instructions were a little hard to follow.
* You said you wanted to do factor reordering by max GDP, but I believe the default fct_reorder, which you used, uses median. 

## File I/O
* Tried multiple ways of exporting/importing data. Nice work!
* No exploration of whether the factors survived after export/import, though.

## Visualization
* Made a figure using color brewer to highlight Canada in a different color. Solid work.

## Writing figures to file
* Saved figures using two different file types. Good work.
* The link to your figure did not work.

## Clean up your repo
* Repo appears pretty clean.

## But I want to do more!
* Solid attempt at this extra task
* Kable of County/Weather does not look correct

## Report your process
* Provided some thoughts about the process. 

## General
* Code was mostly easy to read, but could be improved with better spacing/formatting
* More description of your steps would be helpful to understand your code
* Overall, solid work. Would suggest a Check.

**Comment by abishekarun** [comment]
Hi @maria815 
Congratulations on completing your homeworks successfully.
Here is my peer review of the same:

**Clean up your repo**
Your repo is very clean and nicely organized. One small suggestion would be to 
mention the tasks completed or functions learnt in each assignment's readme 
file.I also think your repo will be tidier if you put the extra files such
as .csv, .txt etc inside a separate folder.

**Factor Management**
- All the required steps are performed. 
- I really liked the idea for the graph you wanted to plot. But I think you have 
reordered based on median of gdpPercap as fct_reorder() takes median by 
default. For min and max calculations I think you need to give min and 
max as third argument in fct_reorder().

**File I/O**
- Different functions for saving the file and reading back the file were explored. 
- One small recommendation would be to explore and identify the difference in these
funcitons as far as retaining the reordered factor levels are concerned.

**Visualization Design**
- The different color palettes and other functions such as setting transparency for
a specific country were explored. 
- One small suggestion would be to explore other theme styles in ggplot and different 
scales such as log scale etc.

**Writing figures to files**
- The figures were saved as png and pdf versions.
- The link to mean_pop seems broken. I think there might be some typo.

**But I want to do more**
- It was nice to see the way new tibble was created and the new function fct_lump() was
explored.
- The table is not formatted nicely.I think you might wanna have a look at it and improve the 
formatting. 
- One suggestion would be to explore more functions in this section. 

**Process Report**
- It was nice to see you mention your thoughts on this homework and tasks completed. 
- One suggestion would be to mention the challenges that you had faced while doing the assignment.

Overall very good job! It is a nice and clear assignment!

Regards
Arun Rajendran

**Comment by derekcho** [comment]
Hi @Maria815, here are some comments about your hw05:

-	The first part of the assignment was well done. You were able to successfully drop Oceania and its associated countries from gapminder.
-	Factor reordering was mostly well done. However, you missed a couple elements for this part even though the end result was good. You are also supposed to explore the effects of arrange by itself on a plot and reordering on a metric other than median (the default that you used)
-	You were successful in writing your output with all three methods taught in class. However, you should also briefly describe the differences between them (write_csv does not preserve factor ordering)
-	Visualization design was well done. Excellent plots!
-	Good work for the additional task!

Your grade will be emailed to you at a later date.
